webserver/pkg/config/config.go

```diff
@@ -116,11 +117,15 @@ func setDefaults(v *viper.Viper) {
 		"logging.output":         "stdout",
 		"logging.include_caller": true,
 
-		"processor.library_base_path":      ".ignore/lib/cuda_learning",
-		"processor.grpc_server_address":    "localhost:60061",
-		"processor.use_grpc_for_processor": true,
+		"processor.library_base_path":   ".ignore/lib/cuda_learning",
+		"processor.grpc_server_address": "localhost:60061",
```

Comment:
this will fail intentionally in the next PR I will fix this with the tailscale name or maybe a treafik route